models/iter8.go

```diff
@@ -0,0 +1,68 @@
+package models
+
+type Iter8Info struct {
+	Enabled bool `json:"enabled"`
+	Permissions ResourcePermissions `json:"permissions"`
+}
+/*
+type ExperimentList struct {
+	Namespace string `json:"namespace"`
+
+	Experiments []ExperimentListItem `json:"experiments"`
+}
+*/
+type ExperimentListItem struct {
```

Comment:
One suggestion, as these classes are public for the "models" package, I think it could be good if Iter8 classes are namespaced with a "Iter8" suffix.
Perhaps Experiment could have not clash, but classes like CriteriaDetail and Metric are generic and other areas of Kiali may use similar naming so a "Iter8CriteriaDetail" o "Iter8Metric" may help on these cases.

In next iterations we can decide if we put all extension under it's own model/subpackage or we maintain this pattern.